apps/settings/js/bluetooth.js

```diff
@@ -664,13 +666,14 @@ navigator.mozL10n.once(function bluetoothSettings() {
         }
       } else {
         // show pair process fail.
-        var msg = _('error-pair-title');
+        var l10nId = 'error-pair-title';
         if (errorMessage === 'Repeated Attempts') {
-          msg = msg + '\n' + _('error-pair-toofast');
+          l10nId = 'error-pair-toofast';
         } else if (errorMessage === 'Authentication Failed') {
-          msg = msg + '\n' + _('error-pair-pincode');
+          l10nId = 'error-pair-pincode';
```

Comment:
ok, I don't even know how to get this code to run :(

The only code piece related to the string `error-pair-title` I can run is using this code: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/panels/bluetooth/panel.js#L593-L599 and it's already properly separating the entities.

The only other place where this l10nId is used is here and I don't know how to trigger this code path.

Can you help me? Can we move it to use `DialogService.alert`?